packages/wallets/src/features/cashier/modules/WithdrawalCrypto/components/WithdrawalCryptoForm/components/WithdrawalCryptoAmountConverter/WithdrawalCryptoAmountConverter.tsx

```diff
@@ -0,0 +1,163 @@
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import { Field, FieldProps, useFormikContext } from 'formik';
+import { useActiveWalletAccount, useCurrencyConfig, useExchangeRate } from '@deriv/api';
+import ArrowBold from '../../../../../../../../public/images/ic-back-arrow.svg';
+import { WalletTextField } from '../../../../../../../../components';
+import type { TForm } from '../../WithdrawalCryptoForm';
+import './WithdrawalCryptoAmountConverter.scss';
+
+type TProps = {
+    activeWallet: ReturnType<typeof useActiveWalletAccount>['data'];
```

Comment:
@aum-deriv just an advice for the future (don't change it now), you can use types from THooks namespace

Reply:
This is done 👍